crates/gosub_shared/src/byte_stream.rs

```diff
@@ -598,12 +597,9 @@ impl LocationHandler {
         if self.cur_location.column > 1 {
             self.cur_location.column -= 1;
             self.cur_location.offset -= 1;
-            return;
-        }
-
-        if self.cur_location.line > 1 {
+        } else if self.cur_location.line > 1 {
             self.cur_location.line -= 1;
-            self.cur_location.column = self.line_endings[&self.cur_location.line];
+            self.cur_location.column = self.column_stack.pop().expect("column_stack is empty");
```

Comment:
Does this work? `pop` removes the last item, shouldn't this be `last`?
Maybe you can replace the expect with a `.unwrap_or(1)`

Reply:
It does work. Column size is pushed when moving to next line.
So last column size must be removed when moving to previous line.
`last` returns last element But doesn't remove element.

To avoid panic, I'll replace `expect` with `.unwrap_or(1)`.

Reply:
Ah, I thought it needs to be removed to actually have a stack of offsets to lines.